gnat2goto/ireps/follow.adb

```diff
@@ -5,6 +5,11 @@ package body Follow is
       while Next /= 0 and then Kind (Next) = I_Symbol_Type loop
          Next := ST (Intern (Get_Identifier (Next))).SymType;
       end loop;
+      if Kind (Next) = I_Bounded_Signedbv_Type then
+         Next :=
```

Comment:
As I recall, the problem we wanted to solve was not at all trivial - for the benefit of future users (and other team members who will have to scratch their heads), I'd suggest notifying that this block is due to requirement that the types of the lhs and rhs of an assignment must being equal - which would be violated for the operation of assignment between a bounded integer and standard integer. This would cause failure in cbmc symex, so we strip the bounds from the type upon export. 
Something to hint at the issue this block solves.